pl_seaweed/init.lua

```diff
@@ -59,22 +62,21 @@ for i in ipairs(algae_list) do
 			local under_node = minetest.get_node(pt.under)
 			local above_node = minetest.get_node(pt.above)
 			local top_node	 = minetest.get_node(top_pos)
-
-			if biome_lib.get_nodedef_field(under_node.name, "buildable_to") then
+			if get_ndef(under_node.name)["buildable_to"] then
 				if under_node.name ~= "default:water_source" then
 					place_pos = pt.under
-				elseif top_node.name ~= "default:water_source"
-							 and biome_lib.get_nodedef_field(top_node.name, "buildable_to") then
+				elseif top_node.name ~= "default:water_source" and get_ndef(top_node.name)["buildable_to"] then
```

Comment:
out of scope i know, but mentioning it. perhaps we want to de hardcode this to check that its a liquid/source and has a certain group or something. because other mods might add water, and river water exists for certain mapgens